.github/workflows/safe_pr_checks.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       - name: Unit Tests
         run: make pytest
 
-      - name: Integration Tests (Access)
-        run: make pytest-integration-access
+      - name: Integration Tests
+        run: make pytest-integration
```

Comment:
so does this still only run access tests?

Reply:
Yes, and by default it will run them for all environments